playwright-e2e/lib/component/dsm/navigation/enums/selectStudyNav.enum.ts

```diff
@@ -16,3 +16,7 @@ export enum Study {
   RAREX = 'RareX',
   SINGULAR = 'Singular'
 }
+
+export enum Miscellaneous {
```

Comment:
To follow the existing design pattern, please create the other file in the same directory, which will have the name: **miscellaneousNav**, and move this part of the code there.

For guidance, this [PR](https://github.com/broadinstitute/ddp-angular/pull/2073/files#diff-a10c040e2ef3ef1a70a165cd73885057a0f0ce0616f8f73e99ee0bb26bfb74c0) might be helpful